package.json

```diff
@@ -96,6 +96,7 @@
     "file-saver": "^2.0.5",
     "ilios-common": "^71.1.1",
     "loader.js": "^4.7.0",
+    "luxon": "^3.0.4",
```

Comment:
wer'e getting Luxon out of common already, so declaring it here again as a dependency is not necessary.

Reply:
ember-auto-import got mad at me if I didn't include this. I think it's because it will eventually be required with embroider.

Reply:
fair enough. moving on then, _to be revisited_.